controller/RequestValidator/InvalidRequestException.php

```diff
@@ -0,0 +1,17 @@
+<?php declare(strict_types=1);
+
+namespace oat\taoPublishing\controller\RequestValidator;
+
+use RuntimeException;
+use Throwable;
+
+class InvalidRequestException extends RuntimeException
+{
+    public function __construct(string $message = "", int $code = 0, Throwable $previous = null) {
```

Comment:
please fix code style,  { move to new line

Reply:
Fixed. :)